### PR TITLE
Feat/geoloc ips

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/web-ca`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-4902
| License          | BSD 3-Clause

## Description

- Remove IP selection if option is not available
- Create component to handle common configuration for webhosting domain

## Related 
- [x] TRANS-25635

### CODE REVIEW

**Comment by marie-j** [comment]
All the feature would need a refactoring (still a wizard, handling everything with `$scope`...) but this is too huge to be handle yet and in this PR

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    label="{{:: 'hosting_multisite_domain_configuration_runtime' | translate }}"
+    data-size="l"
+>
+    <oui-select
+        data-items="$ctrl.runtimes"
+        data-model="$ctrl.runtime"
+        name="runtime"
+        match="name"
+    >
+        <span ng-bind="$item.name"></span>
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html:32]: 👀 On custom elements we tend to drop the `data` prefix but we not used to apply
this rule to the native HTMLElement. Is it intentional?

Comment (marie-j) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.html]: Oopsie ! I think I mixed both rules, I've updated it

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "hosting_multisite_domain_configuration_home": "Dossier racine",
+  "hosting_multisite_domain_configuration_myfolder": "mondossier",
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/translations/Messages_fr_FR.json:3]: Intentional ?

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.component.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import controller from './configuration.controller';
+import template from './configuration.html';
+
+export default {
+  bindings: {
+    cdn: '=',
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.component.js:6]: Just so you know, I hate this, it means I have to go to the `*.component.js` file to know how to use the component, whereas is everything was in `'<'`, we would be so fine 😅

**Comment by marie-j** [comment]
I forgot to move some translations keys so I just updated the new translations file and remove old key

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/translations/Messages_fr_FR.json (hunk 2 above)
Comment (marie-j) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/translations/Messages_fr_FR.json:3]: I've just extracted the translations so component can have its own. No content update made

File: packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.component.js (hunk 3 above)
Comment (marie-j) [reply packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.component.js:6]: It won't work well in the case of a form though 😬

**Comment by marie-j** [comment]
Rebased on `feat/web-ca`

**Comment by marie-j** [comment]
Rebased on feat/web-ca

**Pull request merged**